components/search-result-counter.jsx

```diff
@@ -5,10 +5,14 @@ export default (props) => {
 
```

Comment:
I double checked with Nat - the pill button is only for displaying "help type". For text(keyword) search we can stick with existing design.